Makefile

```diff
@@ -9,3 +9,17 @@ build:
 
 check:
 	GO111MODULE=on ./scripts/check-everything.sh
+
+test: build
+	./cli-experimental --kubeconfig ~/.kube/config apply cmd/test-manifests/hello
+	./cli-experimental --kubeconfig ~/.kube/config apply status cmd/test-manifests/hello
```

Comment:
Do you need to verify the status is as expected?

Reply:
I just added it for manual testing. will remove.